release/src/router/httpd/httpd.c

```diff
@@ -589,7 +595,7 @@ send_headers( int status, char* title, char* extra_header, char* mime_type, int
     (void) fprintf( conn_fp, "x-frame-options: SAMEORIGIN\r\n");
     (void) fprintf( conn_fp, "x-xss-protection: 1; mode=block\r\n");
 
-    if (fromapp != 0){
+    if (fromapp != 0 && !check_mime_type_and_return_true_if_javascript(mime_type)){
```

Comment:
`enum {
	FROM_BROWSER,
	FROM_ASUSROUTER,
	FROM_DUTUtil,
	FROM_ASSIA,
	FROM_IFTTT,
	FROM_ALEXA,
	FROM_WebView,
	FROM_UNKNOWN
};`
`if (fromapp != 0)` this means the request is not from the browser, you can't clear the cache on these devices until you reboot these devices.
`struct mime_handler mime_handlers[]` you can find what you want.

FROM_BROWSER:web browser,such as chrome, firefox, ie
FROM_ASUSROUTER:asus app
FROM_DUTUtil:fcc testing tool or development tool
FROM_ASSIA/FROM_IFTTT/FROM_ALEXA:iot device
FROM_WebView:asus app web browser